### DISCUSSION TITLE
Directional arrows

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#117] [Q&A] [answered]
I noticed that you can no longer see the directional arrows.
You can only see the corners of the arrows....

**Comment by devemux86** [comment, marked as answer]
@Leeb64 can you post a screenshot?
What happens if you zoom?

Route arrows are dynamic, they are generated based on route geometry.
Straight sections have more available space than sections with angles.

Or is a track overlay on top of the route arrows?

**Reply by Leeb64** [reply]
![grafik](https://user-images.githubusercontent.com/120722053/230398860-06b192d8-e486-416e-9c3e-3bec64427bd8.png)
Zoom doesn't matter

**Reply by devemux86** [reply]
Do you mean the red line, the arrows exist, is it a route?

But there is a black line at the top, is it a track overlay?

**Reply by devemux86** [reply]
If the black line is the GPS recording, you can hide it with "Settings | GPS recording | Map".

**Reply by Leeb64** [reply]
The black line is the route.
The street is red.
The arrows are hidden under the ruote

**Reply by devemux86** [reply]
> The arrows are hidden under the route

This is strange, can you post the route file and app settings .pref file?

**Reply by Leeb64** [reply]
My mistake!
I have imported a curvy route -> overlay+route.
The black line covering the arrows is the overlay...
Learned something again...

**Comment by devemux86** [comment]
You could set a different color for the route and track overlay
and select "Dashed line" as track style in routing settings,
so they appear differently on the map.
